references/cli/kube.go

```diff
@@ -0,0 +1,215 @@
+
+	var paths []string
+	for _, file := range opt.files {
+		path := strings.TrimSpace(file)
+		if !slices.Contains(paths, path) {
+			paths = append(paths, path)
+		}
+	}
+	for _, path := range paths {
+		data, err := utils.LoadDataFromPath(cmd.Context(), path, utils.IsJSONOrYAMLFile)
+		if err != nil {
+			return err
+		}
+		opt.filesData = append(opt.filesData, data...)
+	}
+	return nil
+}
+
+// Validate .
+func (opt *KubeApplyOptions) Validate() error {
+	if len(opt.filesData) == 0 {
+		return fmt.Errorf("not file found")
+	}
+	for _, fileData := range opt.filesData {
+		decoder := yaml.NewDecoder(bytes.NewReader(fileData.Data))
+		for {
+			obj := &unstructured.Unstructured{Object: map[string]interface{}{}}
+			err := decoder.Decode(obj.Object)
+			if err != nil {
+				if errors.Is(err, io.EOF) {
+					break
+				}
+				return fmt.Errorf("failed to decode object in %s: %w", fileData.Path, err)
+			}
+			if opt.namespace != "" {
+				obj.SetNamespace(opt.namespace)
+			} else if obj.GetNamespace() == "" {
+				obj.SetNamespace(metav1.NamespaceDefault)
+			}
+			opt.objects = append(opt.objects, obj)
+		}
+	}
+	return nil
+}
+
+// Run .
+func (opt *KubeApplyOptions) Run(f velacmd.Factory, cmd *cobra.Command) error {
+	if opt.dryRun {
+		for i, obj := range opt.objects {
+			if i > 0 {
+				_, _ = fmt.Fprintf(opt.Out, "---\n")
+			}
+			bs, err := yaml.Marshal(obj.Object)
+			if err != nil {
+				return err
+			}
+			_, _ = opt.Out.Write(bs)
+		}
+		return nil
+	}
+	for i, cluster := range opt.clusters {
+		if i > 0 {
+			_, _ = fmt.Fprintf(opt.Out, "\n")
+		}
+		_, _ = fmt.Fprintf(opt.Out, "Apply objects in cluster %s.\n", cluster)
+		ctx := multicluster.ContextWithClusterName(cmd.Context(), cluster)
+		for _, obj := range opt.objects {
+			copiedObj := &unstructured.Unstructured{}
+			bs, err := obj.MarshalJSON()
+			if err != nil {
+				return err
+			}
+			if err = copiedObj.UnmarshalJSON(bs); err != nil {
+				return err
+			}
+			res, err := controllerutil.CreateOrPatch(ctx, f.Client(), copiedObj, nil)
+			if err != nil {
+				return err
+			}
+			key := strings.TrimPrefix(obj.GetNamespace()+"/"+obj.GetName(), "/")
+			_, _ = fmt.Fprintf(opt.Out, "  %s %s %s.\n", obj.GetKind(), key, res)
+		}
+	}
+	return nil
+}
+
+var (
+	kubeApplyLong = templates.LongDesc(i18n.T(`
+		Apply Kubernetes objects in clusters
+
+		Apply Kubernetes objects in multiple clusters. Use --clusters to specify which clusters to
+		apply. If -n/--namespace is used, the original object namespace will be overrode.
+
+		You can use -f/--file to specify the object file to apply. Multiple file inputs are allowed.
+		Directory input and web url input is supported as well.`))
+
+	kubeApplyExample = templates.Examples(i18n.T(`
+		# Apply single object file in managed cluster
+		vela kube apply -f my.yaml --cluster cluster-1
+		
+		# Apply multiple object files in multiple managed clusters
+		vela kube apply -f my-1.yaml -f my-2.yaml --cluster cluster-1 --cluster cluster-2
```

Comment:
```suggestion
		vela kube apply -f my-1.yaml -f my-2.yaml --cluster cluster-1 --cluster cluster-2
```
Is the command supported?

`		vela kube apply -f my-1.yaml -f my-2.yaml --cluster cluster-1 cluster-2
`

Reply:
here support --cluster cluster1,cluster2 OR --cluster cluster1 --cluster cluster2